locales/pt/pt.json

```diff
@@ -280,8 +280,8 @@
     "Increase": "Aumentar",
     "India": "Indio",
     "Indonesia": "Indonesio",
-    "Iran, Islamic Republic Of": "Iraniano",
-    "Iran, Islamic Republic of": "Iran, Islamic Republic of",
+    "Iran, Islamic Republic Of": "República Islâmica do Irão",
+    "Iran, Islamic Republic of": "República Islâmica do Irão ",
```

Comment:
Extra space character at the end of a line.

Reply:
fixed.